actions/utils/deploy/action.yml

```diff
@@ -34,12 +34,17 @@ inputs:
   image_tag:
     required: true
     description: "The image tag to deploy."
+
+  location_load_timeout:
```

Comment:
maybe deploy_timeout? The timeout it will wait for it be deployed by dagster cloud?